watch.json

```diff
@@ -3,7 +3,6 @@
     "include": ["^package\\.json$", "^\\.env$"]
   },
   "restart": {
-    "exclude": ["^lib/"],
```

Comment:
is this on purpose?

Reply:
yeah per this conversation: https://fogcreek.slack.com/archives/CJ1G77E87/p1575562388019200

Reply:
We talked about in #pod-shared-components. Without this line, you have to run `refresh` to see any changes you make. I'm not sure why it was there in the first place.

Reply:
cool cool